dotnet3.5/ACM/DescribeCertificates/DescribeCertificate/DescribeCertificate.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[acm.dotNET.DescribeCertificate]
+
+using Amazon;
+using Amazon.CertificateManager;
+using Amazon.CertificateManager.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace DescribeCertificate
+{
+    class DescribeCertificate
+    {
+        // The following example retrieves and displays the metadate for a
+        // certificate using the AWS Certificate Manager (ACM) service. It
+        // was created using AWS SDK for .NET 3.5 and .NET 5.0.
+
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint ACMRegion = RegionEndpoint.USEast1;
+        private static AmazonCertificateManagerClient _client;
+
+        static void Main(string[] args)
+        {
+            var _client = new Amazon.CertificateManager.AmazonCertificateManagerClient(ACMRegion);
+
+            var describeCertificateReq = new DescribeCertificateRequest();
+            // The ARN used here is just an example. Replace it with the ARN of
+            // a certificate that exists on your account.
+            describeCertificateReq.CertificateArn = "arn:aws:acm:us-east-1:123456789012:certificate/8cfd7dae-9b6a-2d07-92bc-1c3093edb218";
+
+            var certificateDetailResp = DescribeCertificateResponseAsync(client: _client, request: describeCertificateReq);
+            var certificateDetail = certificateDetailResp.Result.Certificate;
+
+            if(certificateDetail is not null)
+            {
+                DisplayCertificateDetails(certificateDetail);
+            }
+
+        }
+
+        /// <summary>
+        /// Displays detailed metadata about a certificate retrieved
+        /// using the AWS Certificate Manager (ACM) service.
+        /// </summary>
+        /// <param name="certificateDetail">The object that contains details
+        /// returned from the call to DescribeCertificateAsync.</param>
+        static void DisplayCertificateDetails(CertificateDetail certificateDetail)
+        {
+            Console.WriteLine("\nCertificate Details: ");
+            Console.WriteLine($"Certificate Domain: {certificateDetail.DomainName}");
+            Console.WriteLine($"Certificate Arn: {certificateDetail.CertificateArn}");
+            Console.WriteLine($"Certificate Subject: {certificateDetail.Subject}");
+            Console.WriteLine($"Certificate Status: {certificateDetail.Status}");
+            foreach (var san in certificateDetail.SubjectAlternativeNames)
+            {
+                Console.WriteLine($"Certificate SubjectAlternativeName: {san}");
+            }
+        }
+
+        /// <summary>
+        /// Calls DescribeCertificateAsyn to retrieve the metadata associated
```

Comment:
A bit verbose. The user can see the call, so I would condense it to:

Retrieves the metadata associated with the ACM service certificate.

Reply:
fixed